tests/IResearch/IResearchViewTest.cpp

```diff
@@ -1553,10 +1536,8 @@ TEST_F(IResearchViewTest, test_properties_internal_request_explicit_version) {
                 "testView" == slice.get("name").copyString());
     EXPECT_TRUE(slice.get("type").isString() &&
                 "arangosearch" == slice.get("type").copyString());
-    EXPECT_TRUE(slice.get("id").isString() &&
-                "101" == slice.get("id").copyString());
-    EXPECT_TRUE(slice.get("planId").isString() &&
-                "101" == slice.get("planId").copyString());
+    EXPECT_TRUE(slice.get("id").isString());
+    EXPECT_TRUE(slice.get("planId").isString());
```

Comment:
Because before my changes even with `isUserAction == true` we created view with `id` and `planId` that were in definition vpack. It's not true anymore, so I adjusted tests